src/components/Discord/Widget.tsx

```diff
@@ -0,0 +1,40 @@
+import DiscordResponse from "@/interfaces/DiscordData";
+import Link from "../Link";
+import User from "./User";
+import { useState } from "react";
+
+interface Props extends Omit<React.HTMLProps<HTMLDivElement>, "value"> {
+  value: DiscordResponse;
+}
+
+export const Widget = ({ value }: Props) => {
+  const [memberOverflow, setMemberOverflow] = useState(false);
+  const { members, presence_count } = value;
+  return (
+    <section className="py-4 bg-dark rounded-md flex flex-col justify-start text-light h-full min-h-[500px]">
+      <h2 className="text-2xl text-gold font-bold px-6 py-4">
+        {presence_count} members online!
+      </h2>
+      <div className="flex grow h-px relative">
+        <div
+          className={`w-full grow flex flex-col gap-1 px-6 overflow-x-hidden overflow-y-auto ${
+            memberOverflow ? "pointer-events-auto" : "pointer-events-none"
+          }`}
+        >
+          {members.map((item) => (
```

Comment:
would it maybe look more attractive if this was sorted so that people playing wow were at the top?

Reply:
That might be something to humor at large, but if there's too few of them it probably feels bad too. So Idk. I think for this PR I'll leave out doing that.